sdks/java/io/redis/src/test/java/org/apache/beam/sdk/io/redis/RedisIOTest.java

```diff
@@ -250,6 +263,72 @@ public void testWriteUsingDECRBY() {
     assertEquals(-1, count);
   }
 
+  @Test
+  public void testWriteStreams() {
+
+    /* test data is 10 keys (stream IDs), each with two entries, each entry having one k/v pair of data */
+    List<String> redisKeys =
+        IntStream.range(0, 9).boxed().map(idx -> UUID.randomUUID().toString()).collect(toList());
```

Comment:
Just fyi, start is inclusive here but end is exclusive 🤷‍♂️ not a very consistent api...

Reply:
lordy.  Fixing. :)